airbyte-webapp-e2e-tests/cypress/pages/replicationPage.ts

```diff
@@ -42,12 +45,28 @@ export const setupDestinationNamespaceSourceFormat = () => {
   cy.get(destinationNamespaceSource).click();
 };
 
-export const selectFullAppendSyncMode = () => {
+export const selectSyncMode = (source: string, dest: string) => {
   cy.get(syncModeDropdown).first().click({ force: true });
 
-  cy.get(`.react-select__menu`)
-    .contains("Append") // it would be nice to select for "Full refresh" is there too
-    .click();
+  cy.get(`.react-select__option`).contains(`Source:${source}| Dest:${dest}`).click();
+};
+
+export const selectCursorField = (value: string) => {
+  cy.get(cursorFieldDropdown).first().click({ force: true });
+
+  cy.get(`.react-select__option`).contains(value).click();
+};
+
+export const checkCursorField = (expectedValue: string) => {
+  cy.get(cursorFieldDropdown).find("p").should("have.text", expectedValue);
+};
+
+export const checkPrimaryKey = (expectedValue: string) => {
+  cy.get(primaryKeyText).should("have.text", expectedValue);
```

Comment:
The same goes for this and other test code which uses `cy.get(someSelector).should("have.text", expectedValue)`: `cy.contains(expectedValue)` or `cy.get(entirePageSectionForSomeSpecificRecord).contains(expectedValue)` is better since it lets us improve or fix the code more easily without needing to edit the test code too.